internal/api/server.go

```diff
@@ -138,22 +106,18 @@ func (s *server) newDashboardRequestHandler() http.HandlerFunc {
 	}
 }
 
+func (s *server) GetVersionInfo(
+	ctx context.Context,
+	req *connect.Request[svcv1alpha1.GetVersionInfoRequest],
+) (*connect.Response[svcv1alpha1.GetVersionInfoResponse], error) {
+	return handler.GetVersionInfoV1Alpha1(version.GetVersion())(ctx, req)
```

Comment:
Do we really need to construct the message every time?

Version information won't change while the server's running, so we may as well do this just once.